test/index-webgl-dependent-tests.js

```diff
@@ -9,5 +9,4 @@ import './modules/core/webgl-context';
 import './modules/core/webgl';
 import './modules/core/core';
 
-// GPGPU is a set of shadertool modules, no WebGL dependencies
```

Comment:
Why?

Reply:
Turns out 1chandu's GPGPU module is not WebGL-independent so I corrected that statement.